nucleus/__init__.py

```diff
@@ -1285,15 +1296,88 @@ def create_dataset_from_dir(
 
         if len(items) == 0:
             print(f"Did not find any items in {dirname}")
-            return None
+            return existing_dataset
```

Comment:
I'm wondering if it would be actually better to create the dataset in this case also. Even though there were not items in the directory, we wanted to create the dataset in the first place.

Reply:
That's a good point, i think the dataset should still be created. I think the original behaviour was misleading.

Reply:
Addressed